go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/siderolabs/talos
 
-go 1.20
+go 1.21
```

Comment:
1.21.0?

Reply:
I'm kinda torn on that, I don't see why we should do a minor version there? unless we depend on something to be exactly 1.21.0, but not 1.21.1 ? I understand this is a compatibility guarantee, or a constraint

Reply:
@DmitriyMV please help

Reply:
@DmitriyMV please help